imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -157,39 +252,37 @@ private ParseResults parseDetailRecords(File file, FileInfo fileInfo) {
 
         } catch (IOException e) {
             String error = String.format("Unable to read %s: %s", file, e.getMessage());
-            reportErrorAndThrow(file.getName(), error);
+            reportAuditAndThrow(file.getName(), error);
         } catch (NoSuchAlgorithmException e) {
             String error = String.format("Unable to compute checksum: %s", e.getMessage());
-            reportErrorAndThrow(file.getName(), error);
+            reportAuditAndThrow(file.getName(), error);
```

Comment:
Should we throw here?  This doesn't seem like a bad request, it seems like an error on our end.